b2/account_info.py

```diff
@@ -141,6 +141,42 @@ def clear_large_file_upload_urls(self, file_id):
         pass
 
 
+class UploadUrlPool(object):
```

Comment:
I'll go ahead and write the test.  It's easier than writing the reason for not writing the test.  :-)